### PR TITLE
Initial release from fork

### PR DESCRIPTION
**Pull request by joneff**
Outstanding issues:

* [x] license ?
* tba

### CODE REVIEW

File: .github/workflows/ci_nightly.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  run:
+    shell: bash
+
+env:
+  CI: true
+
+on:
+  schedule:
+    # Run at midnight every working day
+    - cron: '0 0 * * Mon-Fri'
```
Comment (tsvetomir) [inline .github/workflows/ci_nightly.yml:13]: Do we need a nightly build? The project doesn't seem to have many dependencies so what's expected to change?

File: .github/workflows/merge_develop.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Merge develop to master
```
Comment (tsvetomir) [inline .github/workflows/merge_develop.yml:1]: Do we need a separate workflow for this? "Release stable" also forwards _master_.

**Review by tsvetomir — COMMENTED** [review]
As a minimum, copy the original MIT license and add a copyright notice:

> Copyright © 2021 Progress Software Corporation and/or one of its subsidiaries or affiliates. All rights reserved. (as modified)

**Review by eeasss — APPROVED** [review]

File: .github/workflows/ci_nightly.yml (hunk 1 above)
Comment (joneff) [reply .github/workflows/ci_nightly.yml:13]: You are right. I added a note to build only if the last CI commit is not the same as the current HEAD

File: .github/workflows/merge_develop.yml (hunk 2 above)
Comment (joneff) [reply .github/workflows/merge_develop.yml:1]: On a very few occasions, yes we do. `semantic-release` remembers the branch it was run in. So if we need to change something about the release, we must manually merge develop to master.

**Review by tsvetomir — APPROVED** [review]
Looks good. Added the necessary secrets for publishing.

**Pull request merged**